### PR TITLE
[K3.1] Change language behavior: The administrator has disabled public write access 

### PR DESCRIPTION
**Pull request by 810**

### CODE REVIEW

File: components/com_kunena/site/language/en-GB/en-GB.com_kunena.views.ini [hunk 1]
```diff
@@ -125,6 +125,8 @@ COM_KUNENA_SELECT_CATEGORY="- Select Category -"
 COM_KUNENA_VIEW_DISABLED="The administrator has disabled public write access."
 COM_KUNENA_VIEW_TOPICS_DEFAULT="Topic: %s"
 COM_KUNENA_VIEW_TOPIC_ERROR_LOOP="Internal Error: Loop detected in moved topics!"
+COM_KUNENA_LOGIN_PLEASE="Please %s %s to join the conversation."
+COM_KUNENA_LOGIN_OR="or"
```
Comment (mahagr) [inline components/com_kunena/site/language/en-GB/en-GB.com_kunena.views.ini]: Splitting sentences like this doesn't really work in some languages.

    COM_KUNENA_LOGIN_PLEASE="Please %s to join the conversation."
    COM_KUNENA_LOGIN_X_OR_Y="%s or %s"

Might be better?

Comment (810) [inline components/com_kunena/site/language/en-GB/en-GB.com_kunena.views.ini]: Then i only see the login . Not the create account sentence

**Comment by mahagr** [comment]
I mean with the appropriate changes to the code as well.

File: components/com_kunena/site/language/en-GB/en-GB.com_kunena.views.ini (hunk 1 above)
Comment (mahagr) [inline components/com_kunena/site/language/en-GB/en-GB.com_kunena.views.ini]: I mean with the appropriate changes to the code as well.

**Pull request merged**